week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
```

Comment:
태그는 목적에 맞게 설정해주는 것이 좋습니다. 이 경우, 요소의 내용이 텍스트로만 이루어져 있고 하나의 문단을 구성하므로 p 태그를 사용하는 것이 좋겠습니다.
```suggestion
            <p class="score">
```